customer_service/lib/FileManager/FileGroup/FileGroupHandler.php

```diff
@@ -0,0 +1,221 @@
+                "', @sID);";
+        if (DatabaseManager::mysql_query($queryHolder)) {
+            if (DatabaseManager::mysql_query($query)) {
+                $result = DatabaseManager::mysql_query($querySelectHolder);
+                if ($result) {
+                    $num_results = DatabaseManager::mysql_num_rows($result);
+                    if ($num_results>0) {
+                        $row = DatabaseManager::mysql_fetch_array($result);
+                        if (strcmp($row["s_id"], "")!=0) {
+                            $this->status = $row["s_id"];
+                        } else {
+                            $errorNode = $errorNodeFactory->createPersistenceError(
+                                "Class:FileGroupHandler->persistFileGroup; Empty status returned ",
+                                "Unable to register file group.<br/>Please try again Later"
+                            );
+                            ErrorReporter::addNode($errorNode);
+                        }
+                    } else {
+                        $errorNode = $errorNodeFactory->createPersistenceError(
+                            "Class:FileGroupHandler->persistFileGroup; Empty status returned ",
+                            "Unable to register file group.<br/>Please try again Later"
+                        );
+                        ErrorReporter::addNode($errorNode);
+                    }
+                } else {
+                    $errorNode = $errorNodeFactory->createPersistenceError(
+                        "Class:FileGroupHandler->persistFileGroup; Null resultset: ".
+                        DatabaseManager::mysql_error(),
+                        "Unable to register file group.<br/>Please try again Later"
+                    );
+                    ErrorReporter::addNode($errorNode);
+                }
+            } else {
+                $errorNode = $errorNodeFactory->createPersistenceError(
+                    "Class:FileGroupHandler->persistFileGroup; Query failed: ".
+                    DatabaseManager::mysql_error(),
+                    "Unable to register file group.<br/>Please try again Later"
+                );
+                ErrorReporter::addNode($errorNode);
+            }
+        } else {
+            $errorNode = $errorNodeFactory->createPersistenceError(
+                "Class:FileGroupHandler->persistFileGroup; Holder failed: ".
+                DatabaseManager::mysql_error(),
+                "Unable to register file group.<br/>Please try again Later"
+            );
+            ErrorReporter::addNode($errorNode);
+        }
+        return $this->status;
+    }
+
+    public function getFileGroups($userId)
+    {
+        $errorNodeFactory = new ErrorNodeFactory();
+        $query = "SELECT * FROM FileGroup WHERE vUserID='".$userId."' AND cStatus='0';";
+        $row="";
+        $response="";
+        $error="";
+        $result = DatabaseManager::mysql_query($query);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid using static access to class '\Lib\Database\DatabaseManager' in method 'getFileGroups'.](https://app.codacy.com/gh/BuildForSDG/Team-064-Backend/pullRequest?prid=5616370)